### PR TITLE
Update Kernel rounding documentation

### PR DESCRIPTION
**Pull request by TylerWitt**
Updates Kernel.round/1 to describe what happens on half.

### CODE REVIEW

File: lib/elixir/lib/kernel.ex [hunk 1]
```diff
@@ -835,7 +835,7 @@ defmodule Kernel do
   end
 
   @doc """
-  Rounds a number to the nearest integer.
+  Rounds a number to the nearest integer. Rounds half away from 0.
```
Comment (josevalim) [inline lib/elixir/lib/kernel.ex]:
```suggestion
  Rounds a number to the nearest integer half away from 0.
```

?

Comment (TylerWitt) [reply lib/elixir/lib/kernel.ex]: How about `Rounds a number to the nearest integer using half away from 0.`?

Comment (josevalim) [reply lib/elixir/lib/kernel.ex]: @TylerWitt :+1:

Comment (eksperimental) [reply lib/elixir/lib/kernel.ex]: Same here

Comment (fertapric) [reply lib/elixir/lib/kernel.ex]: What do you think about?

>  If the number is equidistant to two integers, rounds away from zero.

Comment (TylerWitt) [reply lib/elixir/lib/kernel.ex]: Updated. Thanks for the suggestions.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**